openml/datasets/dataset.py

```diff
@@ -173,6 +173,28 @@ def __init__(self, name, description, format=None,
         else:
             self.data_pickle_file = None
 
+    def __str__(self):
+        object_dict = self.__dict__
+        output_str = ''
+        name = '\n%-14s: %s\n' % ("Name", object_dict['name'])
+        version = '%-14s: %s\n' % ("Version", object_dict['version'])
+        format = '%-14s: %s\n' % ("Format", object_dict['format'])
+        date = '%-14s: %s\n' % ("Upload Date", object_dict['upload_date'].replace('T', ' '))
+        licence = '%-14s: %s\n' % ("Licence", object_dict['licence'])
+        d_url = '%-14s: %s\n' % ("Download URL", object_dict['url'])
+        base_url = 'https://www.openml.org/d/'
```

Comment:
Could you please use the server URL given by `openml.config.server`? In case the user works against the test server, this URL would be wrong.

Reply:
I am unsure of what to do here.
Shall I change the ` base_url = 'https://www.openml.org/d/'` to ` base_url = 'https://test.openml.org/d/'` ? 
However,  [https://www.openml.org/d/31](https://www.openml.org/d/31)  and [https://test.openml.org/d/31](https://test.openml.org/d/31) don't point to the same resource.

Reply:
Understood. Made the changes.